src/Microsoft.IdentityModel.Clients.ActiveDirectory/AuthenticationContext.cs

```diff
@@ -234,9 +234,8 @@ public string KeychainSecurityGroup
         /// <param name="resource">Identifier of the target resource that is the recipient of the requested token.</param>
         /// <param name="clientId">Identifier of the client requesting the token.</param>
         /// <returns>It contains Device Code, its expiration time, User Code.</returns>
-#if iOS
-        [Obsolete("This device profile API should only be used on text-only devices, and not on this target platform which is offering an interactive authentication experience. For details please see https://aka.ms/AdalNetConfFlows")]
-#endif
+        [Obsolete("Microsoft.IdentityModel.Clients.ActiveDirectory has been deprecated and replaced with Microsoft.Identity.Client. " +
+            "To migrate an app using Device Code flow, see https://learn.microsoft.com/en-us/azure/active-directory/develop/msal-net-migration-public-client?tabs=devicecode")]
```

Comment:
would it be better to use aka.ms links which we can change if needed? If so, happy to help create them.

Reply:
lol, that's what a elow.

We already have https://aka.ms/adal-net-to-msal-net that maps to https://learn.microsoft.com/azure/active-directory/develop/msal-net-migration.

I think we should have:
- https://aka.ms/adal-to-msal-net/device-code
- https://aka.ms/adal-to-msal-net/interactive
- https://aka.ms/adal-to-msal-net/daemon
- https://aka.ms/adal-to-msal-net/obo
- https://aka.ms/adal-to-msal-net/web-app

Reply:
Sure, let me do this.

Reply:
(and yes, I did remove the en-us from the target urls)

Reply:
I would like to have the web app and web API links so that we can change them later (For the moment it would go to MSAL or IDWeb, but this will change in the future when Id.Web 2.0 exists).
There will be ASP.NET Core, OWIN, SDK scenarios.

Reply:
This has been done by @jennyf19 , resolving the comment.